src/lib/components/Framework/Page.svelte

```diff
@@ -141,6 +141,7 @@
 	header .logo {
 		height: var(--page-logo-height);
 		width: var(--page-logo-width);
+		display: var(--page-logo-display);
```

Comment:
Are we expecting content editors to know about display:block vs display:inline etc?
Or was the thinking that this is a boolean?  In which case perhaps "--page-logo-show" just to distinguish it.  I'm leaning towards not needing this at all - if you put an image link in then you want to show an image.

Reply:
they would know about css yes. someone would need to set it up, if the customer cannot, the  we can do it for a cost.